scripts/build_renewable_profiles.py

```diff
@@ -194,20 +194,75 @@
 import time
 
 import atlite
+import country_converter as coco
 import geopandas as gpd
 import numpy as np
+import pandas as pd
 import progressbar as pgb
 import xarray as xr
-from _helpers import configure_logging, sets_path_to_root
+from _helpers import configure_logging, read_csv_nafix, sets_path_to_root
+from add_electricity import load_powerplants
 from pypsa.geo import haversine
 from shapely.geometry import LineString
-from shapely.ops import unary_union
+
+cc = coco.CountryConverter()
 
 logger = logging.getLogger(__name__)
 
 COPERNICUS_CRS = "EPSG:4326"
 GEBCO_CRS = "EPSG:4326"
 
+
+def get_eia_annual_hydro_generation(fn, countries):
```

Comment:
Do you think we should move this and the below function in future in a dedicated rule as done in https://github.com/PyPSA/pypsa-eur/blob/master/scripts/build_hydro_profile.py

Reply:
Further, should we add a docstring with a link to the data? Should we also do that for `get_hydro_capacity_annual_hydro_generation()`?

Reply:
> Do you think we should move this and the below function in future in a dedicated rule as done in https://github.com/PyPSA/pypsa-eur/blob/master/scripts/build_hydro_profile.py

Yeah, I was exactly thinking about that as well. We may think how we want the build_renewable profiles. Whether we want them to be split by resource or only hydro/all_the_rest.

> Further, should we add a docstring with a link to the data? Should we also do that for `get_hydro_capacity_annual_hydro_generation()`?

That's kind of duplicated with respect to the data (see link in the eia csv), but it can be useful